pinot-broker/src/main/java/org/apache/pinot/broker/broker/BasicAuthAccessControlFactory.java

```diff
@@ -55,10 +57,16 @@ public BasicAuthAccessControlFactory() {
     // left blank
   }
 
+  @Override
   public void init(PinotConfiguration configuration) {
     _accessControl = new BasicAuthAccessControl(BasicAuthUtils.extractBasicAuthPrincipals(configuration, PREFIX));
   }
 
+  @Override
+  public void init(ZkHelixPropertyStore<ZNRecord> propertyStore) {
+  }
+
+  @Override
```

Comment:
same here.

Reply:
same

Reply:
Sure. I have simply moved the implementation of init(PinotConfiguration, ZkHelixPropertyStore) to the AccessControlFactory parent class and declared it non-abstract. 
So this class (and similar ones)  remain unchanged.